site/sfguides/src/data_collaboration_native_app/data_collaboration_native_app.md

```diff
@@ -0,0 +1,630 @@
+- No customer data leaves Snowbank's Snowflake Account
+- The Native App runs within the Snowbank Account. Since Snowflake has been approved for use internally, it is not subject to longer onboarding and cybersecurity checks
+
+The advantages for Zamboni are:
+- Simple deployment. Zamboni can simply share the model using a similar listing paradigm as sharing a dataset.
+- Reduced compliance risk. Zamboni's security and compliance team are similarly pleased in being able to satisfy the business needs without needing Snowbank's data hosted on their servers
+- Increased Margin. Zamboni are now able to share their proprietary logic safely, without having to provision infrastructure to run the model on behalf of Snowbank.
+- Speed to Market. Zamboni can simply distribute the model via the Native App framework in a private listing. They no longer need to set up data pipelines. They can also build once, and distribute to lots of different customers by leveraging Snowflake's multitenancy framework.
+
+What is different in this scenario compared to Part One, is that it is no longer a bi-directional share of data. Now Zamboni becomes the provider of an Application, and Snowbank becomes the consumer.
+
+### Dataset Details
+
+We will use a publicly available dataset to train our ML model for the purposes of this demo. The dataset contains aggregated profile features for each customer at each statement date. Features are anonymized and normalized, and fall into the following general categories:
+
+D_* = Delinquency variables
+S_* = Spend variables
+P_* = Payment variables
+B_* = Balance variables
+R_* = Risk variables
+
+### Dataset Citation
+
+Addison Howard, AritraAmex, Di Xu, Hossein Vashani, inversion, Negin, Sohier Dane. (2022). American Express - Default Prediction. Kaggle. https://kaggle.com/competitions/amex-default-prediction
+
+<!-- ------------------------ -->
+## Set up
+Duration: 10
+
+Navigate to the [Snowflake Trial Landing Page](https://signup.snowflake.com/?utm_cta=quickstarts_). Follow the prompts to create a Snowflake Account. You should receive an email to activate your trial account.
+
+Navigate to a new worksheet and execute the following commands to create a second account in the Organisation. The second account will be the account we share the model to (Snowbank). In trial accounts we cannot share the Native App externally, although the same process can be followed to share the app externally if you have a non-trial account.
+
+```SQL
+USE ROLE ORGADMIN;
+
+CREATE ACCOUNT SNOWBANK
+  ADMIN_NAME = ADMIN
+  ADMIN_PASSWORD = 'ENTER PASSWORD HERE'
+  EMAIL = 'ENTER YOUR EMAIL HERE'
+  EDITION = ENTERPRISE;
+```
+Note down the account locator and the url from the output of the command above for use in later steps
+
+<!-- ------------------------ -->
+
+## Provider Account (Zamboni) - Set Up
```

Comment:
@sfc-gh-tbuchhorn Rename this to "Setup" and add "Provider Account (Zamboni)" sub-header underneath.